integration/container/stats_test.go

```diff
@@ -0,0 +1,53 @@
+package container
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/gotestyourself/gotestyourself/poll"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAPIStatsContainerGetMemoryLimit(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
```

Comment:
The old test had `memoryLimitSupport`, do we still need that? It looks like it's just 

```go
skip.If(t, !testEnv.DaemonInfo.MemoryLimit)
```

Reply:
Done.